cartographer/mapping_2d/sparse_pose_graph/optimization_problem.h

```diff
@@ -65,12 +65,18 @@ class OptimizationProblem {
   void AddImuData(int trajectory_id, const sensor::ImuData& imu_data);
   void AddOdometerData(int trajectory_id,
                        const sensor::OdometryData& odometry_data);
+  void InsertTrajectoryNode(const mapping::NodeId& node_id, common::Time time,
```

Comment:
Consistency: `Insert...` after `Add...`, same in the `.cc` file to match 3D?

Reply:
Removed and reordered.